src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -21,6 +21,7 @@ internal abstract class WorkerProcess : IWorkerProcess, IDisposable
         private readonly IWorkerConsoleLogSource _consoleLogSource;
         private readonly IScriptEventManager _eventManager;
         private readonly IMetricsLogger _metricsLogger;
+        private readonly int processExitTimeoutInMilliseconds = 60000;
```

Comment:
>private readonly int processExitTimeoutInMilliseconds = 60000; [](start = 8, length = 62)

Added an arbitrary timeout here. Would love to get feedback if you had something else in mind

Reply:
1 minute might be too long.  lets change this to 1sec